charmhelpers/contrib/openstack/context.py

```diff
@@ -738,11 +742,22 @@ def __call__(self):
                                                           laddr)])
                 }
                 for rid in relation_ids('cluster'):
-                    for unit in sorted(related_units(rid)):
-                        # API Charms will need to set {addr_type}-address with
-                        # get_relation_ip(addr_type)
+                    # API Charms will need to set {addr_type}-address with
+                    # get_relation_ip(addr_type)
+                    if enable_lb is True:
```

Comment:
I believe I used this syntax and it was not working. This is why I changed it to

`if enabl_lb is True:`

I can test it one more time, however.

Reply:
Interesting.  So essentially, if `if enable_lb:` didn't pass the tests, but `if enable_lb is True` did pass the tests, then a "false" value is actually 'truthy'.  i.e. a string like `"false"` or an object.  False values are only `0, False, "", None`.  It would probably be a good idea to add an `assert` (for testing purposes) in the code to ensure that the type of the value is what you are expecting.

e.g.

```python
assert type(enable_lb) is bool
```